### PR TITLE
Add reminder should persist in all 11.3.x releases

### PR DESCRIPTION
**Pull request by cokiengchiara**
**Note for future 11.3.x releases:** Until upstream external-dns issue is fixed, please include this note and the one above in all future 11.3.x releases.

### CODE REVIEW

**Comment by sslavic** [comment]
Few days ago I've reported the mentioned issue upstream https://github.com/kubernetes-sigs/external-dns/issues/1617
Should we
- link to it in the release notes?
- put it on our backlog to work on a fix ourselves?

**Comment by pipo02mix** [comment]
> Should we
> - link to it in the release notes?
> - put it on our backlog to work on a fix ourselves?

Both?

File: aws/v11.3.3/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ automatically renewed, which would cause expired certificates to show your site
 
 **Note for SEs:** This release contains an external-dns fix introduced in [11.3.2](https://github.com/giantswarm/releases/blob/master/aws/v11.3.2/release-notes.md). It requires manual intervention for cluster upgrades in China to work. When upgrading a cluster, existing ingress A+TXT record sets do not get replaced with CNAME+TXT record sets even when external-dns is configured with CNAMEs as preferred. After upgrading, delete the ingress A+TXT record sets. external-dns will then automatically create CNAME+TXT record sets.
 
+**Note for future 11.3.x releases:** Until upstream external-dns issue is fixed, please include this note and the one above in all future 11.3.x releases.
```
Comment (cokiengchiara) [inline aws/v11.3.3/release-notes.md:10]:
```suggestion
**Note for future 11.3.x releases:** Until [upstream external-dns issue](https://github.com/kubernetes-sigs/external-dns/issues/1617) is fixed, please include this note and the one above in all future 11.3.x releases.
```

**Comment by cokiengchiara** [comment]
> > Should we
> > 
> > * link to it in the release notes?
> > * put it on our backlog to work on a fix ourselves?
> 
> Both?

@sslavic @pipo02mix thanks! I linked to it in release notes. Unless it takes 2mins 🤪 I think we don't fix it ourselves, since it only affects one customer and only in China...

**Pull request merged**